storm-client/src/jvm/org/apache/storm/grouping/LoadAwareShuffleGrouping.java

```diff
@@ -47,13 +47,14 @@
     private Random random;
     private volatile int[] prepareChoices;
     private AtomicInteger current;
-    private Scope currentScope;
+    @VisibleForTesting
+    LocalityScope currentScope;
```

Comment:
I would keep this as private and add a `getCurrentScope` function for test only with `@VisibleForTesting` annotation.